src/components/Tree/readme.md

```diff
@@ -94,17 +94,17 @@
 
 ```js
     const data = [
-        { label: 'Tree Item', icon: <FileIcon /> },
-        { label: 'Tree Item', icon: <FileIcon /> },
-        { label: 'Tree Item', icon: <FileIcon /> },
+        { label: 'Tree Item',isChecked: false, icon: <FileIcon /> },
+        { label: 'Tree Item',isChecked: false, icon: <FileIcon /> },
+        { label: 'Tree Item',isChecked: false, icon: <FileIcon /> },
         {
             label: 'Tree Branch',
             icon: <FolderCloseIcon />,
             isExpanded: true,
-            isChecked: false,
+            isChecked: 'indeterminate',
             children: [
                 { label: 'Tree Item', isChecked: false },
-                { label: 'Tree Item', isChecked: false },
+                { label: 'Tree Item', isChecked: true },
```

Comment:
I think we should make this example works, e.g:
- select all when checking the parent checkbox
- deselect all when unchecking the parent checkbox
- set to indeterminate the parent checkbox when only some and not all child checkbox are selected